server/fleet/cron_stats.go

```diff
@@ -0,0 +1,44 @@
+package fleet
+
+import "time"
+
+// CronStats represents statistics recorded in connection with a named set of jobs (sometimes
+// referred to as a "cron" or "schedule"). Each record represents a separate "run" of the named job set.
```

Comment:
Should we consolidate on a single name? I would vote to drop the usage of "cron" everywhere. We seem to mostly use "schedule" now.